packages/containers/src/ConfirmDialog/ConfirmDialog.test.js

```diff
@@ -1,18 +1,36 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 import { fromJS, Map } from 'immutable';
-import { store, Provider } from '@talend/react-cmf/lib/mock';
+import mock, { store, Provider } from '@talend/react-cmf/lib/mock';
 
 import Container from './ConfirmDialog.container';
-import Connected from './ConfirmDialog.connect';
-
+import Connected, { mapStateToProps } from './ConfirmDialog.connect';
 
 import { showConfirmDialog, hideConfirmDialog } from './showHideConfirmDialog';
 
-jest.mock(
-	'@talend/react-components',
-	() => ({ ConfirmDialog: props => (<div className="tc-confirm-dialog" {...props} />) })
-);
+jest.mock('@talend/react-components', () => ({
+	ConfirmDialog: props => <div className="tc-confirm-dialog" {...props} />,
+}));
+
+describe('Action.mapStateToProps', () => {
```

Comment:
`Connected ConfirmDialog mapStateToProps `
Or better, move that into `describe('Connected ConfirmDialog')`